apps/i18n/common/en_us.json

```diff
@@ -2071,6 +2071,7 @@
   "standardsReportHeader": "Class Standards Report",
   "standardsReportNoUnpluggedLessons": "There are no unplugged lessons in this course.",
   "standardsReportLessonLengthInfo": "*Lessons in this course offer between 45 and 65 minutes of instruction",
+  "startBlocksTemplateWarning": "WARNING: You are editing start blocks for a level with a template. Any start blocks defined here will be overwritten by the template.",
```

Comment:
>Any start blocks defined here will be overwritten by the template.

Strictly, I think this is only true when there are no start blocks defined in the template. It's possible to define per-level start blocks in a templated progression, but we don't think that you should do that.

Possible alternative:
> "WARNING: You are editing start blocks for a level with a template. Start blocks should be defined in the template level."

Reply:
Should there still be a notification that if start blocks are defined in the template, per-level start blocks are over-written?

Reply:
I actually don't think we should say "overwritten" anywhere because that implies to me that we'll be erasing something. If start blocks are defined in both places, we simply ignore any that are defined in the non-template levels.

Reply:
Then should we remind levelbuilders that the level start blocks will be ignored if there are start blocks defined in the template?

Reply:
We do this already near the field on level edit pages where templates are added:
<img width="975" alt="image" src="https://user-images.githubusercontent.com/43474485/217305183-9078d521-d8b7-4081-851f-cd905be34af6.png">

I think this banner is primarily meant to be a reminder that you're editing start blocks (rather than working in the level as a student), and secondarily as an extra warning to be careful if you're using a template. As long as the message we add to the banner is accurate and helpful, I don't think it has to explain every detail about the behavior.

Reply:
👍 I like your text Mike, I could only make the string so long without it spilling onto a second line and making the banner too big. Hopefully the clarification on the edit page is enough. This is mostly to give a reminder.